src/System.Runtime.Serialization.Xml/tests/Performance/PerformanceTestsCommon.cs

```diff
@@ -0,0 +1,207 @@
+﻿using System.IO;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Xunit.Performance;
+using System.Xml;
+using SerializationTypes;
+
+namespace System.Runtime.Serialization
+{
+    #region Performance tests configuration types
+
+    public enum SerializerType
+    {
+        DataContractSerializer,
+        DataContractJsonSerializer,
+        XmlSerializer,
+        JsonNet
+    }
+
+    public enum TestType
+    {
+        SimpleType,
+        ISerializable,
+        Dictionary,
+        ListOfInt,
+        String,
+        ByteArray,
+        XmlElement
+    }
+
+    public interface IPerfTestSerializer
+    {
+        void Init(object obj);
+        void Serialize(object obj, Stream s);
+        void Deserialize(Stream s);
+    }
+
+    public abstract class SerializerFactory
```

Comment:
Why use an interface for IPerfTestSerializer and an abstract class for SerializerFactory? I would make them both interfaces as abstract classes are for when you need some implementation to exist in a common base class. Whether you prefer interfaces of base abstract classes, you should be consistent.